### PR TITLE
Extended the previous "artisan" alias to take in arguments...

### PR DESCRIPTION
**Pull request by jjsee**
Also made it shorter by adding the "art" alias for quick access.

### CODE REVIEW

File: src/stubs/aliases [hunk 1]
```diff
@@ -44,3 +44,7 @@ function serve-proxy() {
         echo "  serve-proxy domain port"
     fi
 }
+
+function artisan() {
+    php artisan $1 $2
```
Comment (taylorotwell) [inline src/stubs/aliases]: I believe there is a way in Bash to just expand out all arguments instead of limiting it to the first 2.

**Comment by taylorotwell** [comment]
This can be changed in your own aliases file I believe.

**Comment by jjsee** [comment]
@taylorotwell There is currently an alias for php artisan that does not work with arguments (on line 6). This is basically to fix that so that it would accept arguments. 

Or if the direction is to keep this in our own aliases file, then perhaps that line should be removed? 

Either way I will be happy to submit a new PR to either update the artisan alias as per your feedback or to remove it entirely.

**Comment by taylorotwell** [comment]
Sure feel free to update it here.

**Comment by jjsee** [comment]
@taylorotwell Done. The artisan() function will now include all arguments instead of just the first two.

**Pull request merged**